cluster/images/kubemark/BUILD

```diff
@@ -0,0 +1,37 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
```

Comment:
don't need this anymore

Reply:
done (thought that was from update-bazel.sh)